eng/CodeChecks.ps1

```diff
@@ -46,6 +46,11 @@ function Invoke-Block([scriptblock]$cmd) {
 }
 
 try {
+    Write-Host "Downloading files"
+    Invoke-Block {
+        & $PSScriptRoot\DownloadSharedSource.ps1 @script:PSBoundParameters
```

Comment:
To run scripts, you don't need the call operator (used for external executables). Instead, you just source the script using the dot operator.
```suggestion
        . $PSScriptRoot\DownloadSharedSource.ps1 @script:PSBoundParameters
```

Same for the same generate.ps1 call below.

Reply:
But I explicitly don't want the side-effects of those scripts execution.